src/appengine/libs/handler.py

```diff
@@ -158,11 +158,11 @@ def get_access_token(verification_code):
 
     See: https://developers.google.com/identity/protocols/OAuth2InstalledApp
   """
```

Comment:
Can you also see that reproduce_tool_client_id gets auto-added to whitelisted_oauth_client_ids, so that it is not needed to be listed there.

Reply:
Done